__tests__/app.test.js

```diff
@@ -314,3 +314,58 @@ describe("GET /api/users", () => {
       .then(({ body }) => expect(body.msg).toBe("Not Found"));
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("200: sucessfully adds a comment to the given article_id", () => {
+    return request(app)
+      .post("/api/articles/6/comments")
+      .expect(201)
+      .send({
+        username: "hello kitty",
+        body: "Hello Kitty is a girl in a costume",
+      })
+      .then(({ body }) => {
+        expect(body.comment.body).toBe("Hello Kitty is a girl in a costume");
+      });
+  });
```

Comment:
This is a good start to this test but it should include all properties we are expecting on the comment object, instead of just the body.